kernels/volk/volk_32f_stddev_and_mean_32f_x2.h

```diff
@@ -175,8 +175,8 @@ static inline void volk_32f_stddev_and_mean_32f_x2_neon(float* stddev,
 
     const float* in_ptr = inputBuffer;
 
-    __VOLK_ATTR_ALIGNED(16) float SumLocal[8] = { 0.f };
-    __VOLK_ATTR_ALIGNED(16) float SquareSumLocal[8] = { 0.f };
+    __VOLK_ATTR_ALIGNED(32) float SumLocal[8] = { 0.f };
+    __VOLK_ATTR_ALIGNED(32) float SquareSumLocal[8] = { 0.f };
```

Comment:
Why does the alignment need to be twice as much as expected? You doubled the other values as well.

Reply:
Because of temporary confusion on my part. But I fixed it and added some alignment directives for NEON as well. I think this is correct now.

Reply:
I'd assume we need 16byte alignment for NEON according to the [docs](https://developer.arm.com/documentation/ddi0344/f/unaligned-data-and-mixed-endian-data-support/unaligned-data-access-support/neon-data-alignment).